### PR TITLE
 fix: population update, table scrape fix 

### PR DESCRIPTION
**Pull request by ArnavD74**
Updated populations for locations without populations, and fixed the table to omit the last 2 roles.

### CODE REVIEW

File: src/events/crawler/scrapers/IND/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      const state = parse.string($tr.find('td:nth-child(2)').text());
+
+      const data = {
+        state,
+        cases: parse.number($tr.find('td:nth-child(3)').text()),
+        deaths: parse.number($tr.find('td:nth-child(6)').text()),
+        recovered: parse.number($tr.find('td:nth-child(5)').text()),
+        population: populationState[state]
+      };
+      if (rules.isAcceptable(data, null, this._reject)) {
```
Comment (lazd) [inline src/events/crawler/scrapers/IND/index.js]: Looks like the rules are no longer present, you can remove this.

File: src/events/crawler/scrapers/IND/index.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  country: 'IND',
+  url: 'https://www.mohfw.gov.in/',
+  type: 'table',
+  aggregate: 'state',
+
+  async scraper() {
+    this.url = 'https://www.mohfw.gov.in/';
+    const $ = await fetch.page(this.url);
+
+    const $table = $('table[style="font-weight:bold"]');
```
Comment (lazd) [inline src/events/crawler/scrapers/IND/index.js]: It looks like it's in a div with id="cases", shouldn't we select based on that, not font-weight?
```suggestion
    const $table = $('#cases table');
```

File: src/events/crawler/scrapers/IND/index.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    $('table tr')
+      .slice(-2)
```
Comment (lazd) [inline src/events/crawler/scrapers/IND/index.js]: This could get brittle if they decide to remove that row, instead, why don't you check for the `colspan` property on the first `td` inside of the table?

File: src/events/crawler/scrapers/IND/index.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      .remove();
+    const $trs = $table.find('tbody > tr');
+    const regions = [];
+
+    $trs.each((index, tr) => {
+      const $tr = $(tr);
+
```
Comment (lazd) [inline src/events/crawler/scrapers/IND/index.js]:
```suggestion
      if ($tr.find('td').first().attr('colspan')) {
        continue;
      }
```

File: src/events/crawler/scrapers/IND/index.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  aggregate: 'state',
+
+  async scraper() {
+    this.url = 'https://www.mohfw.gov.in/';
+    const $ = await fetch.page(this.url);
+
+    const $table = $('table[style="font-weight:bold"]');
+    $('table tr')
+      .slice(-2)
+      .remove();
```
Comment (lazd) [inline src/events/crawler/scrapers/IND/index.js]: There's no need to call `remove()` here.
```suggestion
```

File: src/events/crawler/lib/maintainers.js [hunk 6]
```diff
@@ -42,7 +42,6 @@ const maintainers = {
     state: 'Victoria',
     city: 'Melbourne',
     flag: '🇦🇺'
-
```
Comment (lazd) [inline src/events/crawler/lib/maintainers.js]: Please try not modify unrelated files.

**Comment by lazd** [comment]
I went ahead and merged this one in manually, making sure to set you as the author.

Shukriya, JAI HIND!

**Comment by ArnavD74** [comment]
> 
> 
> I went ahead and merged this one in manually, making sure to set you as the author.
> 
> Shukriya, JAI HIND!

Thanks
Originally, the table was more primitive and did not have an id, and I had not meant to commit the changes I made to the other files.
